project-cards-without-creator-link

From: Bryan Larsen <bryan@larsen.st>

One thing you'll notice is that the project cards have a link to the project owner. In general that's quite useful, but in this context it doesn't make much sense. DRYML is very good at favouring context over consistency -- we can remove that link very easily:

SHOW_PATCH
---
 app/views/front/index.dryml |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/app/views/front/index.dryml b/app/views/front/index.dryml
index eadb165..2ef4f9a 100644
--- a/app/views/front/index.dryml
+++ b/app/views/front/index.dryml
@@ -25,7 +25,7 @@
 
     <section class="content-body" if="&logged_in?">
       <h3>Your Projects</h3>
-      <collection:projects with="&current_user"/>
+      <collection:projects with="&current_user"><card without-creator-link/></collection>
     </section>
   </content:>
 
